### ISSUE TITLE
Multi-volume works

### ISSUE THREAD

**Issue by seabelis** [internetarchive/openlibrary#1586]
It would be nice to have a way to link individual volumes of a single work into a record under the work as an edition as well as each volume having individual records. The form as it is now requests information for a single physical volume, so when you have one work split into three, it is not possible to specify details accurately, i.e. cover, pagination, identifiers, etc. for each volume.
For example, https://openlibrary.org/works/OL17915213W/Harry_Potter_and_the_Order_of_the_Phoenix_Chapters_24-30

**Comment by LeadSongDog** [comment]
Yes.  #561 relates. However the part/section/issue/vol.no. should probably be somehow specific to one edition.

**Comment by seabelis** [comment]
This is slightly different than just works of a series. I mean part of a work. But in the end, I think it boils down to the same treatment. So if Brothers Karamazov is broken up into four physical volumes, there would be a record for each physical volume, but a way to represent the work as a whole under the main Brothers Karamazov work where the individual volumes would be listed together and linked from the record.

**Comment by LeadSongDog** [comment]
What I was getting at was that various publishers' editions do not necessarily partition the work in the same way, even in a single language. Over time, the Encyclopedia Britannica had quite a range of volume counts per edition. Facet filtering of popular works by year and language is growing more and more urgently necessary.

**Comment by seabelis** [comment]
I can understand some arguments for combining partial works with whole works, but that is not so practical for someone who wants to use the reading log. That will mark all partial works as read or to be read simultaneously.

**Comment by LeadSongDog** [comment]
The answer to that is for the log to record the edition read, which is better anyhow.

**Comment by seabelis** [comment]
> The answer to that is for the log to record the edition read, which is better anyhow. There’s a difference between having read a work in the original or in translation after all.

Agreed, that would be better, though I do still see a problem equating volume with edition.

**Comment by LeadSongDog** [comment]
Sorry, that was sloppy of me to not specify that I was using "edition" in the OL sense, not the wider sense.
 
[rant on]
Putting a different colour of cover on it makes the "edition" different here.

Hereabouts, an "edition" has a meaning closer to FRBR's "manifestation", a situation with which I've never really been happy. In FRBR-speak, the parts of a trilogy are referential works, connected with "is continued in part by" or "continues in part" relations. Here at OL, chaos reigns: either the trilogy or the part may be treated as the creative work. I don't think that will change any time soon. 

Personally, I draw the line at calling a microfilm or a scan a distinct edition, but others will even do that. To me, that's just a reprint in a different manifestation. I don't consider that slapping a different publisher's city on the title page rises to the creative threshold deserving the name "editing".
[rant off]

**Comment by seabelis** [comment]
Related to https://github.com/internetarchive/openlibrary/issues/1808

**Comment by guyjeangilles** [comment]
defaulting to assigning @mekarpeles per slack discussions

**Comment by agmckee** [comment]
I keep running into this. 19th century works were frequently distributed in volumes, e.g. 2 volumes. What exactly is the best practice with this presently?

For example, these two works:
https://openlibrary.org/works/OL13121608W
https://openlibrary.org/works/OL13121609W

Are really just one work in two volumes.

**Comment by bicolino34** [comment]
@agmckee currently, you can add this info to Series Name field in Publishing Info section on edtion editing tab. Something like: Name of book, Volume 1
![image](https://user-images.githubusercontent.com/58372007/185631700-e72bc4a2-1370-4417-b410-2936fed84d46.png)
@cdrini should soon start working on book series which should improve current approach. They are tracked in issue #6718

**Comment by agmckee** [comment]
Thanks. I did see #6718, and I did consider use of the "Series Name" but I wasn't sure that this was the best approach as it seems hackish and not the right approach (the example I gave is not a series but two volumes published at the same time as part of the same work).

**Comment by bicolino34** [comment]
I think, it may be a good idea to also add "Volume 1" and "Volume 2" to work's titles to differentiate them from work page that contains editions that were published as one whole rather than two separate volumes. But I'm not sure if my opinion is correct. @seabelis should know better. She's more experienced than I am.

**Comment by tfmorris** [comment]
Any workaround using the current data model is just going to be a hack, but I think a better fit would be to have a single work with each of the two volumes listed as a separate "edition." Still not correct, but better, I think, than creating imaginary works. (It's a single work published in two volumes, as it says on the title page).

**Comment by LeadSongDog** [comment]
Agreed. I have sometimes shown volume in the pagination, such as "3v., v2: 234p. ill."

**Comment by seabelis** [comment]
@agmckee There's no one-size-fits-all answer to this question. In this case, I've added Volume I/II to the titles. In cases where the volumes themselves have titles, you would usually use the volume title at the work level.
